docs/api-reference/lights/sunlight.md

```diff
@@ -0,0 +1,36 @@
+# TODO need update when new light source interface arrives
+ 
+# Sunlight (Experimental)
+
+Sunlight is a directional light source simulating the sun. Sun position calculations are based on [article](http://aa.quae.nl/en/reken/zonpositie.html) and inspired by [SunCalc](https://www.npmjs.com/package/suncalc). 
+
+## Usage 
+
+Create an ambient light source.
+
+```js
+import {_Sunlight as Sunlight} from '@deck.gl/core';
+
+new Sunlight({
+  timestamp: 1554927200000, 
+  color: [255, 0, 0],
+  intensity: 1
+});
+```
+
+## constructor
+
+The constructor for the `Sunlight` class. Use this to create a new `Sunlight`.
+
+```js
+const sunlight = new Sunlight({timestamp, color, intensity});
+```
+
+#### Parameters
```

Comment:
Not 100% sure what conventions deck.gl docs currently follow but I would not add another header here, doesn't help with readability.

setProps is not documented, but see my comment below...